### PR TITLE
Implement grant rework apis except make suggestion

### PR DESCRIPTION
**Pull request by evq**

### CODE REVIEW

File: migrations/0001_initial_schema.up.sql [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+create table issuers (
+  promotion_id uuid not null references promotions(id),
+  cohort text not null,
+  public_key text not null,
+  primary key (promotion_id, cohort)
+);
+
+create table wallets (
+  id uuid primary key not null,
+  -- created_at timestamp with time zone not null default current_timestamp,
```
Comment (maikelmclauflin) [inline migrations/0001_initial_schema.up.sql]: do we not want this info?

File: docker-compose.yml [hunk 2]
```diff
@@ -19,15 +14,55 @@ services:
     ports:
       - "3333:3333"
     environment:
-      - "REDIS_URL=${REDIS_URL-redis://grant-redis:6379}"
+      - "DATABASE_URL=postgres://grants:password@postgres/grants?sslmode=disable"
+      - "LEDGER_SERVER=https://ledger-staging.mercury.basicattentiontoken.org"
+      - "REDIS_URL=${REDIS_URL-redis://redis:6379}"
```
Comment (maikelmclauflin) [inline docker-compose.yml]: does it make sense to remove the option to swap this out. just to avoid situations like those that we saw with ledger.

Comment (evq) [reply docker-compose.yml]: good idea - let's avoid that

File: migrations/0001_initial_schema.up.sql (hunk 1 above)
Comment (evq) [reply migrations/0001_initial_schema.up.sql]: it's not super useful unless we're synchronizing with the ledger held value. let me think on it more - maybe we should do that before moving forward

**Pull request merged**